src/transformers/models/auto/tokenization_auto.py

```diff
@@ -78,6 +78,7 @@
             ("blenderbot", ("BlenderbotTokenizer", "BlenderbotTokenizerFast")),
             ("blenderbot-small", ("BlenderbotSmallTokenizer", None)),
             ("blip", ("BertTokenizer", "BertTokenizerFast" if is_tokenizers_available() else None)),
+            ("blip-2", ("BertTokenizer", "BertTokenizerFast" if is_tokenizers_available() else None)),
```

Comment:
BLIP-2 supports any `AutoTokenizer`, as it can be coupled with any language model.

Hence I'm not sure what to put here

Reply:
It should be the default when nothing in specified in the config.

Reply:
Ok, will update to `GPT2Tokenizer`